### PR TITLE
1.7.7 is ready for translation

### PR DESCRIPTION
**Pull request by Junebyun**

### CODE REVIEW

**Comment by ttoine** [comment]
June can you please check that I did not make a mistake?
then it will be ready to publish it tomorrow.

**Comment by Junebyun** [comment]
@ttoine Everything seems fine, thank you!

**Comment by ttoine** [comment]
so let's publish it tomorrow morning. we will just have to edit the time.

**Comment by Junebyun** [comment]
@ttoine Cool! At time should we publish it?

File: news/_posts/translation/177-is-ready-for-translation.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  "1.7.7 is ready for translation"
+subtitle: "On your marks, set, translate :fire:"
+date:   2020-06-05 09:10:11
```
Comment (ttoine) [inline news/_posts/translation/177-is-ready-for-translation.md]:
```suggestion
date:   2020-06-05 11:00:00
```

**Comment by ttoine** [comment]
let's do it now.

**Comment by ttoine** [comment]
thanks June for this article.

**Pull request merged**